runtime/lua/vim/treesitter.lua

```diff
@@ -5,10 +5,10 @@ local Range = require('vim.treesitter._range')
 ---@type table<integer,LanguageTree>
 local parsers = setmetatable({}, { __mode = 'v' })
 
----@class TreesitterModule
----@field highlighter TSHighlighter
----@field query TSQueryModule
----@field language TSLanguageModule
+---@class vim.TreesitterModule
```

Comment:
```suggestion
---@class vim.treesitter
```

Reply:
Should I do the same for highlighter/query/language module and rename them to vim.treesitter.*?